addons/portal/static/src/js/portal.js

```diff
@@ -283,6 +283,93 @@ publicWidget.registry.portalSecurity = publicWidget.Widget.extend({
 
 });
 
+/**
+ * Defining what happens when you click the "Log out from all devices" 
+ * on the "/my/security" page.
+ */
+publicWidget.registry.RevokeSessionsButton = publicWidget.Widget.extend({
```

Comment:
Why not adding a new file so that we avoid bloating existing files ? Moreover it helps understanding the various features and widget when looking at a module content :)

Reply:
@tde-banana-odoo 
If I create a new file with the contents of my feature, maybe should I also move all the features displayed on the /my/security page so that we have a portal_security.js file instead of a portal_revoke_all_devices.js with only a few lines ?